theSrc/scripts/Legend.js

```diff
@@ -117,7 +112,7 @@ class Legend {
       pt: movedPt[0],
       lab: movedLab[0],
       anchor: 'start',
```

Comment:
So now we determine the number of decimals from the data values? It doesn't use `xFormat` or `yFormat`?

Reply:
Correct. I thought it would be easier to do this than to parse `xFormat` and ``yFormat` which I'm not familiar with. Are they always specified?

Reply:
It appears that axis numbers are automatically formatted based on input data when decimal places are not specified, which is most of the time.

Reply:
I think that axis label decimal settings are not actually useful, as plotly chooses the best one for the situation. What would be useful are decimal settings for tooltips and dragged points. This is not something we have to change now, but something to bear in mind in the future if we redesign this for ngviz.

Reply:
Ok, I think this is something we can do later if we get requests for it